Client/InfluxDBClient.cs

```diff
@@ -285,19 +285,21 @@ public class InfluxDBClient : IInfluxDBClient
         /// </summary>
         /// <param name="host">The URL of the InfluxDB server.</param>
         /// <param name="token">The authentication token for accessing the InfluxDB server.</param>
+        /// <param name="authScheme">Token authentications scheme.</param>
         /// <param name="organization">The organization name to be used for operations.</param>
         /// <param name="database">The database to be used for InfluxDB operations.</param>
         /// <example>
         /// using var client = new InfluxDBClient("https://us-east-1-1.aws.cloud2.influxdata.com", "my-token", "my-org", "my-database");
         /// </example>
-        public InfluxDBClient(string host, string token, string? organization = null,
+        public InfluxDBClient(string host, string token, string? authScheme = null, string? organization = null,
```

Comment:
This is potential breaking change, plese update README.md with similar description as is https://github.com/influxdata/influxdb-client-csharp/blob/master/CHANGELOG.md#migration-notice

Reply:
README or CHANGELO?

Reply:
Update README.md or CHANGELOG.md?